docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.3'
+
+x-db-settings: &db-settings
+  pull_policy: always
+  environment:
+    - DATABASE_NAME=callisto
+    - DATABASE_USERNAME=dev
+    - DATABASE_PASSWORD=dev
+    - DATABASE_ENDPOINT=postgres
+    - DATABASE_PORT=5432
+    - HOME=/Users/1000
+    - MAVEN_USER_HOME=/Users/1000/.m2
+  networks:
+    - callisto
+
+x-liquibase-settings: &liquibase-setting
```

Comment:
Same comment as above regarding there being no resuse. simpler to put in the liquibase service

Reply:
sure, got it fixed locally. Will push ASAP.